README.md

```diff
@@ -19,3 +19,74 @@ Github Pages only runs in safe mode, preventing the usage of [symlinks](https://
 1. Copy the contents of `node_modules/@patternfly/patternfly` to `_includes/patternfly`.
 1. Delete `_includes/patternfly/assets` and `_includes/patternfly/icons` as they are not needed.
 1. Commit the changes.
+
+## Deployment
+
+### OKD
+
```

Comment:
Thinking a one-liner to the headings will help future xyz dev understand why these have two different approaches:

```
OKD
The default templates for OKD are built into the oauth server template.go files via the following method:
OCP
The OCP branded templates take advantage of the override mechanism by providing customized templates via secrets via the following method:
```